### PR TITLE
Update Breadcrumb API

### PR DESCRIPTION
**Pull request by almedina-ms**
Update breadcrumb API based on internal discussions

### CODE REVIEW

File: active/Breadcrumb/Breadcrumb.md [hunk 1]
```diff
@@ -114,6 +115,18 @@ unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
     static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
```
Comment (RBrid) [inline active/Breadcrumb/Breadcrumb.md:115]: DropdownItemTemplateProperty --> DropDownItemTemplateProperty

**Review by RBrid — APPROVED** [review]
We just also need DropDownItemTemplateProperty.

File: active/Breadcrumb/Breadcrumb.md (hunk 1 above)
Comment (almedina-ms) [reply active/Breadcrumb/Breadcrumb.md:115]: Fixed

**Review by kat-y — APPROVED** [review]

**Comment by kat-y** [comment]
Merged these API updates to avoid merge conflicts, see https://github.com/microsoft/microsoft-ui-xaml-specs/pull/102

**Pull request merged**